beatrix/pom.xml

```diff
@@ -63,6 +63,17 @@
             <artifactId>guice</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <exclusions>
```

Comment:
Should we have a scope `test` ?

Reply:
Technically, yes, but Maven complains about it (as it is a transitive dependency for another non-test library).